.rustfmt.toml

```diff
@@ -2,6 +2,7 @@ unstable_features = true
 imports_layout = "Vertical"
 imports_granularity = "Crate"
 comment_width = 120
+# STYLE&CONVENTION: formatting influences content
```

Comment:
Syntax and layout enforces an influence over the code written. This influence can be specially negative in verbose languages, leading the programmer to take decisions focusing in fighting back the verbosity instead of more valuable factors.

These 3 rustfmt customizations might be causing an extra verbosity burden to the code of this project:
- `imports_layout = "Vertical"`: Rust code tends to have many imports, and stacking all of them vertically is a very inefficient way to lay them out, leading programmers to use glob imports (`*`) to circumvent that (which should really, really not be used except for very limited exceptions)